src/detection_win.cpp

```diff
@@ -382,7 +382,7 @@ void BuildInitialDeviceList() {
 		item->deviceState = DeviceState_Connect;
 
 		DWORD DataT;
-		DllSetupDiGetDeviceRegistryProperty(hDevInfo, pspDevInfoData, SPDRP_HARDWAREID, &DataT, (PBYTE)buf, MAX_PATH, &nSize);
+		DllSetupDiGetDeviceRegistryProperty(hDevInfo, pspDevInfoData, SPDRP_LOCATION_INFORMATION, &DataT, (PBYTE)buf, MAX_PATH, &nSize);
```

Comment:
Can you give an actual example of what `SPDRP_HARDWAREID` vs `SPDRP_LOCATION_INFORMATION` will be?

Reply:
Some docs here, https://msdn.microsoft.com/en-us/library/windows/hardware/ff551967(v=vs.85).aspx